src/UiPath.Workflow/Activities/ScriptingJitCompiler.cs

```diff
@@ -118,6 +119,23 @@ public class VbJitCompiler : ScriptingJitCompiler
 
     protected override string CreateExpressionCode(string types, string names, string code) =>
         $"Public Shared Function CreateExpression() As Expression(Of Func(Of {types}))\nReturn Function({names}) ({code})\nEnd Function";
+
+    protected override IEnumerable<string> GetIdentifiers(SyntaxTree syntaxTree)
+    {
+        return syntaxTree.GetRoot().DescendantNodesAndSelf().Where(n => ValidSyntaxNode(n))
+                         .Select(n => n.ToString()).Distinct().ToArray();
+
+        bool ValidSyntaxNode(SyntaxNode node)
+        {
+            if (node.RawKind != IdentifierKind)
+                return false;
+
+            if (node.Parent.RawKind == (int)SyntaxKind.SimpleMemberAccessExpression && (node.Parent as MemberAccessExpressionSyntax)?.Name?.ToString() == node.ToString())
```

Comment:
This looks like a case-sensitive comparison. How is it working?

Reply:
It seems that it is enough to make sure of the uniqueness of the parameters. Please see the latest change. :)